packages/config/src/options.ts

```diff
@@ -181,260 +178,245 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     validation: validate.isPlainObject,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'requestTimeout',
     defaultValue: 5000,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'resolvedNodePath',
     defaultValue: null,
     validation: validate.isString,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'resolvedNodeVersion',
     defaultValue: null,
     validation: validate.isString,
-    canUpdateDuringTestTime: false,
   }, {
     name: 'responseTimeout',
     defaultValue: 30000,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'retries',
     defaultValue: {
       runMode: 0,
       openMode: 0,
     },
     validation: validate.isValidRetriesConfig,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'screenshotOnRunFailure',
     defaultValue: true,
     validation: validate.isBoolean,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'screenshotsFolder',
     defaultValue: 'cypress/screenshots',
     validation: validate.isStringOrFalse,
     isFolder: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'slowTestThreshold',
     defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? 250 : 10000,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'scrollBehavior',
     defaultValue: 'top',
     validation: validate.isOneOf('center', 'top', 'bottom', 'nearest', false),
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
   }, {
     name: 'supportFile',
     defaultValue: (options: Record<string, any> = {}) => options.testingType === 'component' ? 'cypress/support/component.{js,jsx,ts,tsx}' : 'cypress/support/e2e.{js,jsx,ts,tsx}',
     validation: validate.isStringOrFalse,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'supportFolder',
     defaultValue: false,
     validation: validate.isStringOrFalse,
     isFolder: true,
-    canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
   }, {
     name: 'taskTimeout',
     defaultValue: 60000,
     validation: validate.isNumber,
-    canUpdateDuringTestTime: true,
+    overrideLevels: ALL_OVERRIDE_LEVELS,
+  }, {
+    name: 'testIsolation',
+    // TODO: When experimentalSessionAndOrigin is removed and released as GA,
+    // update the defaultValue from 'default' to 'strict' and
+    // update this code to remove the check/override specific to enable
+    // strict by default when experimentalSessionAndOrigin=true
+    defaultValue: 'default',
+    validation: validate.isOneOf('default', 'strict'),
```

Comment:
I went 'default', however writing this change summary, wondering if I should have gone 'standard'?

Reply:
I agree the value name shouldn't be 'default', especially since once session/origin goes GA, it will no longer be the default option. I think we should name it 'legacy'. That's more-or-less what it is, since it's what Cypress historically has supported and 'strict' is the way forward. It's also a wink-wink, nudge-nudge that setting it to 'legacy' is not the ideal way to go.

Reply:
I changed to legacy and it just didn't feel right since this behavior will be around for awhile. I went with 'lax' instead. what do you think?

Reply:
okay...overruled....legacy it is!